EcoSoapBank/mockImpactStatsByPropertyId.json

```diff
@@ -0,0 +1,13 @@
+
+{
+    "data": {
+        "impactStatsByPropertyId": {
+            "soapRecycled": 1,
+            "linensRecycled": 2,
+            "bottlesRecycled": 3,
+            "paperRecycled": 4,
+            "peopleServed": 5,
+            "womenEmployed": 6
+        }
+    }
+}
```

Comment:
I assume you will be writing the tests for decoding models then?

Reply:
Correct. I'm working on that next.